src/Data/Char/Unicode/Internal.purs

```diff
@@ -105,17 +105,20 @@ instance showCharBlock :: Show CharBlock where
                          <> " }"
 
 
-gencatZP :: Int
-gencatZP = 67108864
+gencatPF :: Int
```

Comment:
I believe the unicode file gets updated regularly with new code points.  These changes are completely okay.